demo/index.min.js

```diff
@@ -2650,7 +2650,7 @@ class BaseInput extends r {
    */
   handleClickClear() {
     this.inputElement.value = "";
-    this.value = undefined;
+    this.value = "";
```

Comment:
I think it's from a change for another work or there should be a change on `base-input.js` for this.

Reply:
This is compiled code. The fact that this is changed means a "build" was not done as part of an earlier PR.